src/generate.jl

```diff
@@ -1,5 +1,10 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
+function generate_deprecated(varargs...; kwargs...)
+    Base.depwarn("Pkg.generate is deprecated. Please use PkgTemplates.jl instead.", Core.Typeof(generate).name.mt.name)
```

Comment:
What's `Core.Typeof(generate).name.mt.name`?

Reply:
It's how the deprecate macro gets the symbol for the function being deprecated.

But we are not using the deprecate macro. We are calling depwarn directly. So we can just hardcode the symbol.

Reply:
@KristofferC Take a look at the new code. Can this be resolved?